app/views/curation_concerns/generic_files/upload/_alerts.html.erb

```diff
@@ -0,0 +1,19 @@
+
+<div class="alert alert-info hide" id="success">
+  You have successfully uploaded some of your files.  Either continue to upload or edit use the links below to abandon the rest of your added files.
+  <p>
+  <%= link_to 'Add Descriptions', edit_polymorphic_path([main_app, :curation_concerns, @presenter]), :class=>'btn' %>
```

Comment:
Oh, thanks for reminding me, I need to convince Princeton all :hash: :rocket: all the time.